src/OrchardCore.Modules/OrchardCore.Media/Views/Admin/MediaApplication.cshtml

```diff
@@ -1,7 +1,25 @@
+@using Microsoft.Extensions.Configuration
+@using OrchardCore.Environment.Shell.Configuration
+
+@inject  IShellConfiguration ShellConfiguration
+
 @{
     Layout = null;
 }
 
+@functions{
+    public int MaxSizeLimit
```

Comment:
We can use an `IOrchardHelper` extension method instead

Reply:
So I would have imagined just a variable at the top would be sufficent? and (probably) more in keeping with the Orchard Core Razor code.

**Note** I am moving the media configuration to `IOptions<MediaOptions>` in the blob PR, but that's not stopping this at all.